jeremy2918/tsseract
# Connect post to backend

# [Connecting posts to backend](https://trello.com/c/7J4j84ab/49-i-want-to-connect-the-posts-endpoint-to-the-front-end-allowing-the-user-to-create-and-save-a-post)


This PR adds the required changes in the `withAuth` high-order component so it works properly, handling the case where the `req` object has many cookies and it has to find the app token. It also (and mainly) adds functionality to save the post that a user is creating to the database.

## Testing Steps

- Pull and checkout to this branch.
- `npm install`
- `npm run tsc`
- `npm test` All tests should be passing successfully.
- `npm run watch`
- Log in and go to `create-post`
- Write a sample post and click `Publish`, if the post passes the validation test, you should have successfully created a post from the client.



## UI Changes
No UI Changes
